src/test/e2e/tests/validateAzureTreeView.test.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect, test } from '../baseTest';
+import { APICenter, Timeout, VSCode } from '../utils/constants';
+import VscodeOperator from '../utils/vscodeOperator';
+
+test('validate azure tree view', async ({ workbox }) => {
+    //set test timeout
+    test.setTimeout(120000);
+
+    await workbox.waitForTimeout(Timeout.PREPARE_TEST);
+    // wait API Center extension installed on VS Code.
+    expect(await VscodeOperator.isSideTabItemExist(workbox, "API Center")).toBeTruthy();
+    await VscodeOperator.activeSideTab(workbox, VSCode.TAB_API_CENTER, Timeout.PREPARE_EXT);
+
+    //expand and validate tree items
+    await VscodeOperator.clickTreeItemChildLinkByText(workbox, "Teams Cloud - E2E Testing with TTL = 1 Days", "Teams Cloud - E2E Testing with TTL = 1 Days");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "apicentertest001")).toBeTruthy();
+    await VscodeOperator.clickTreeItem(workbox, "apicentertest001");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "APIs")).toBeTruthy();
+    await VscodeOperator.clickTreeItemChildLinkByText(workbox, "APIs", "APIs");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "callback-example")).toBeTruthy();
+    await VscodeOperator.clickTreeItem(workbox, "callback-example");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "Versions")).toBeTruthy();
```

Comment:
same here, better to check Deployments exists.

Reply:
got it, added